tests/util/test_wb_logger.py

```diff
@@ -113,6 +113,11 @@ def test_wandb_output_format():
         {"_step": 0, "foo": 42, "fizz": 12},
         {"_step": 3, "fizz": 21},
     ]
+
+    with pytest.raises(ValueError, match=r"wandb.Video accepts a file path.*"):
```

Comment:
Testing invalid input error handling is good but it's a bit odd we're not also testing that it does the right thing with a valid input?

Reply:
Testing the basic saving features already exists in a previous test (since manual video saving is already supported) -- I just added this test since it was in the original PR and figured it couldn't hurt.